src/main/python/fine_tuning/xfold.py

```diff
@@ -106,8 +106,9 @@ def tune(self,verbose):
                                 if param not in training_data:
                                     training_data[param] = .0
                                 training_data[param] += fold_performance[param]
+                        # sort based on performance first, then use filenames(x[0]) to break ties
```

Comment:
The sorting will be in descending order for both performance and file names, right?
Can you please add this to the comment?

Reply:
done